modules/git/tag.go

```diff
@@ -64,7 +64,7 @@ l:
 			}
 			nextline += eol + 1
 		case eol == 0:
-			tag.Message = string(data[nextline+1 : len(data)-1])
+			tag.Message = string(data[nextline+1:])
```

Comment:
I think the panic is because of `nextline > len(data) -1`. It should not be resolved.

Reply:
At this point, can we use TrimSpace?

Reply:
this is fine if we add LimitReaders to the appropriate places.